lk2nd/smp/clock-msm8953.c

```diff
@@ -0,0 +1,35 @@
+// SPDX-License-Identifier: BSD-3-Clause
+/* Copyright (c) 2022, Stephan Gerhold <stephan@gerhold.net> */
+
+#include <board.h>
+#include <boot.h>
+#include <clock.h>
+#include <platform/iomap.h>
+#include <reg.h>
+
+#define MHZ				1000000
+
+static bool platform_has_second_cluster(void)
+{
+	switch (board_platform_id()) {
+		case MSM8953:
+		case SDM632:
+		case SDA632:
+			return true;
+		default:
+			return false;
+	};
+}
+
+static int lk2nd_8953_clock_setup(void *dtb, const char *cmdline,
+				  enum boot_type boot_type)
+{
+
```

Comment:
I'd add a check here to skip this for downstream and lk2nd like in clock-msm8916.c:

```c
	/* Downstream sets this itself */
	if (boot_type & (BOOT_DOWNSTREAM | BOOT_LK2ND))
		return 0;